draft-ietf-taps-interface.md

```diff
@@ -503,44 +503,49 @@ implemented over UDP.
 This property specifies whether the application needs to use a transport
 protocol that ensures that all data is received on the other side without
 corruption. This also entails being notified when a Connection is closed or
-aborted. The recommended default is to enable Reliable Data Transfer.
+aborted. The recommended default is to Require Reliable Data Transfer.
+
+### Preservation of Message Boundaries {#prop-boundaries}
+
+This property specifies whether the application needs to use a transport
+protocol that preserves message boundaries. The recommended default
+is to Prefer Preservation of Message Boundaries.
 
 ### Configure per-Message reliability {#prop-partially-reliable}
 
 This property specifies whether an application considers it useful to indicate
 its reliability requirements on a per-Message basis. This property applies to
-Connections and Connection Groups. The recommended default is to not have this
+Connections and Connection Groups. The recommended default is to Prefer this
```

Comment:
Really? I mean I'm all for a set of defaults that drives SCTP (or future PR-QUIC) usage, but I don't think most applications care about this.

Reply:
Prefer only means that you'd rather get a protocol that can do this if possible. It's harmless to request. A part of "not caring" is perhaps also the fact that this is so often not available?

Reply:
I think this discussion around defaults, here and at other places in the document, indicates that we need to have a "not defined" option as a default for several of them. 
The indicated preferences should be used to make the selection of the transport. This should not be influenced by various parameters that are not relevant to the application. I would not like to get or not get a transport becuase of a default setting of a parameter that is not relevant for my application. On the other hand, if I prefer a protocol that provides for instance partial relaiability I would like that to have some impact on what is selected. So it should not be "harmless" to request something.

Reply:
I totally agree!  Actually I think the issue is - I tried to make a choice between the defined preference levels, and couldn't find a reasonable "neutral" one: the middle one is defined as:

> Ignore  - Cancel any system default preference for this property

... but "Cancel any system default" hardly makes sense for a recommended default. I think we want the middle preference level to mean "don't care".  (which I'd prefer over "not defined" because, well, there must be a default value, so what does "not defined" mean?)

So I suggest:

> Neutral - No preference for or against this property

Reply:
yep, let's rename Ignore to Neutral.

Reply:
Done (and applied to the (now two separate) checksum-related Selection properties.

Reply:
Sorry, but I actually think Ignore is a better name. Perhaps better defined as  

> Ignore - Not used during protocol/path selection

> 

And we should use it :-)

Reply:
Sigh  :)   but I agree, the problem that I had was the "cancel any system default preference" description, not the name. I'm fine with going back to Ignore. Will do in the next commit.

Reply:
Done, I changed it back to "Ignore" but kept the newer description.